rdmo/management/assets/js/components/common/Links.js

```diff
@@ -179,18 +179,19 @@ ExtendLink.propTypes = {
   onClick: PropTypes.func.isRequired
 }
 
-const CodeLink = ({ className, uri, onClick }) => {
+const CodeLink = ({ className, uri, onClick, order }) => {
   return (
-    <Link onClick={onClick}>
+    <><Link onClick={onClick}>
       <code className={className}>{uri}</code>
-    </Link>
+    </Link>{order !== undefined && order !== null ? <code className="code-order ng-binding">{order}</code> : null}</>
```

Comment:
I think there should be a `{' '}` between the URI and the order.

Reply:
You can use lodash's `!isEmpty(order)` here (`import isEmpty from 'lodash/isEmpty'`).

Reply:
> You can use lodash's `!isEmpty(order)` here (`import isEmpty from 'lodash/isEmpty'`).

Unfortunately `isEmpty` does not work with numbers

Reply:
right! Its `isNil` https://lodash.com/docs/#isNil